tests/iframe-test.js

```diff
@@ -1,9 +1,9 @@
 //These this needs to be revisited.
-QUnit.test( "Dropkick closes from click event in parent document when running inside an iframe", function( assert ) {
+// QUnit.test( "Dropkick closes from click event in parent document when running inside an iframe", function( assert ) {
 //   var dk = new Dropkick("#normal_select");
 
 //   dk.open();
 //   assert.equal(dk.isOpen, true);
 //   $(".body").trigger("click");
 //   assert.equal(dk.isOpen, false);
-});
+// });

```

Comment:
This probably can get moved into the main test file and create a helper that builds / cleans up iframe creation